kalite/distributed/static/js/distributed/topics/router.js

```diff
@@ -25,12 +33,7 @@ ChannelRouter = Backbone.Router.extend({
             });
             this.channel = channel;
         }
-        splat = splat || "/";
-        if (splat.indexOf("/", splat.length - 1)==-1) {
-            splat += "/";
-            this.navigate(Backbone.history.getFragment() + "/", {replace: true});
-        }
-        this.control_view.navigate_paths(splat.split("/"));
+        this.navigate_splat(splat);
```

Comment:
This method was previously being overwritten by the navigate_channel method in TopicChannelRouter (ChannelRouter was not being used anywhere, by anything, so it seemed pointless to leave it lying around).